evennia/settings_default.py

```diff
@@ -538,6 +560,10 @@
 PROTOTYPEFUNC_MODULES = ["evennia.utils.prototypefuncs",
                          "server.conf.prototypefuncs"]
 
+# Module holding validator functions. functions in later modules will
+# override those in earlier ones.
+VALIDFUNC_MODULES = ['evennia.utils.validfuncs', ]
```

Comment:
"validfunc" makes this sound like another of Evennia's custom functions like protfunc, inlinefunc etc - making them feel a bit more custom than they need to be. I think `OPTIONS_VALIDATOR_MODULES` is clearer (and also helps group the settings-name with the `OPTIONS_ACCOUNT_DEFAULT` suggested above).